src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -561,6 +562,18 @@ class ArnoldRenderer : public IECoreScenePreview::Renderer
 				}
 				return;
 			}
+			else if( name == g_logFileNameOptionName )
+			{
+				if( value == NULL )
+				{
+					AiMsgSetLogFileName( "" );
```

Comment:
Arnold actually has two logs that are controlled independently - one goes to the file and one goes to the console. So this just turns off the file one, leaving the console one unaffected. I'm a bit undecided as to how to expose the other logging options, but this one seemed straightforward and pretty useful.

It'd be good to chat about how we do expose the other stuff though :

- The logging verbosity is controlled at a low level using a bitmask of AI_LOG_INFO, AI_LOG_STATS etc - you can see the full list in the [API documentation](https://trac.solidangle.com/arnoldpedia/chrome/site/Arnold-4.0.14.0/doc/api/group__ai__msg.html#ga8a3bb774002173067560d320b04124ab). But then kick exposes this as a verbosity integer ranging from 0 to 6. How do we want to expose it? I don't really like the 0..6 approach because it's not really obvious what you're gonna get. But the other approach has a lot of options (12 by my count).
- The logging verbosity is controlled separately for file and console logs. Should we also expose that?
- We can also attach our own handler for Arnold messages. Should we use that to redirect the console log into our own IECore::MessageHandler system, and then from there on into a ConsoleWindow type thing in Gaffer? At which point we'd have filtering by log level anyway.